packages/smooth_app/lib/background/background_task_crop.dart

```diff
@@ -198,7 +198,7 @@ class BackgroundTaskCrop extends BackgroundTaskUpload {
     final LocalDatabase localDatabase,
     final bool success,
   ) async {
-    localDatabase.upToDate.terminate(uniqueId);
+    await super.postExecute(localDatabase, success);
```

Comment:
If you say await here, wouldn't be better to also have an async call for the delete just below?
(Just a suggestion)

Reply:
> If you say await here, wouldn't be better to also have an async call for the delete just below? (Just a suggestion)

That would probably work the same, wouldn't it? Especially here, where we delete just one file in a background task.